recipes/libqasm/all/conanfile.py

```diff
@@ -0,0 +1,113 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMakeToolchain, CMakeDeps, CMake, cmake_layout
+from conan.tools.files import get
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.scm import Version
+import os
+
+class LibqasmConan(ConanFile):
+    name = "libqasm"
+
+    # Optional metadata
+    license = "Apache-2.0"
+    homepage = "https://github.com/QuTech-Delft/libqasm"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Library to parse cQASM files"
+    topics = ("code generation", "parser", "compiler", "quantum compilation", "quantum simulation")
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "build_python": [True, False],
+        "build_tests": [True, False],
+        "compat": [True, False],
+        "tree_gen_build_tests": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "build_python": False,
+        "build_tests": False,
+        "compat": False,
+        "tree_gen_build_tests": False
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 23
```

Comment:
```suggestion
        return 11
```

Seems like the min cppstd is 11 https://github.com/QuTech-Delft/libqasm/blob/develop/CMakeLists.txt#L8?

Reply:
`libqasm` builds with C++11 at the moment, but I'm going to be working on a evolution of the library, in order to support the version 3.0 of the `cqasm` language and, for that evolution, I had thought of using C++23 features. However, since Conan Center seems to build binaries only up to C++20, I'm finally going to use C++20, at least for the near future.

Reply:
Let me talk with the team on when you can expect C++23 configurations to be available and I'll get back to you in a few days, please feel free to use it for your project!

Reply:
@RubenRBS  I think having C++23 in 2023 should be expected

Reply:
I've left this set to `20` in the end. Does Conan Center already support `23`?